### ISSUE TITLE
Analysis of contributors over time: Active vs Drifting vs Gone

### ISSUE THREAD

**Issue by cdolfi** [sandiego-rh/sandiego#138]

**Comment by cdolfi** [comment]
@JamesKunstle is this being worked on right now?

**Comment by JamesKunstle** [comment]
@cdolfi yeah finishing some stuff for it tomorrow. today was dev-ops based but tomorrow I'm going to finish the initial range requirement that you have in your other visualizations to finish up that half-baked PR.
